src/material-experimental/mdc-form-field/public-api.ts

```diff
@@ -6,17 +6,21 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-export {
-  MAT_FORM_FIELD,
-  MatFormFieldControl,
-  getMatFormFieldDuplicatedHintError,
-  getMatFormFieldMissingControlError,
-} from '@angular/material/form-field';
-
 export * from './directives/label';
 export * from './directives/error';
 export * from './directives/hint';
 export * from './directives/prefix';
 export * from './directives/suffix';
 export * from './form-field';
 export * from './module';
+
+export {
+  MAT_FORM_FIELD,
+  MatFormFieldControl,
+  getMatFormFieldDuplicatedHintError,
```

Comment:
Hmm, these don't have an underscore but they are `@docs-private`. They really shouldn't have been part of the public API... @jelbourn do you think we can skip re-exporting `@docs-private` stuff?

Reply:
I think that we have things like this in a bunch of packages, but we just have to live with it and eventually deprecate them.

Reply:
Same, if we decide to keep these we should at least move them under a TODO to get rid of them later

Reply:
Yeah, at this point anything without an underscore would have to go through the normal process. I would like to explore removing "library internal" symbols from our npm packages via transformation at some point, but that's probably not going to happen soon.